recipes/icu/meta.yaml

```diff
@@ -35,20 +36,23 @@ test:
        "libicuuc"
      ] %}
     {% for each_lib in libs %}
+    - test -f $PREFIX/lib/{{ each_lib }}.a                                                 # [unix]
     - test -f $PREFIX/lib/{{ each_lib }}.{{ version }}.dylib                               # [osx]
     - test -f $PREFIX/lib/{{ each_lib }}.so.{{ version }}                                  # [linux]
     {% endfor %}
-    - derb --help             # [unix]
-    - genbrk --help           # [unix]
-    - gencfu --help           # [unix]
-    - gencnval --help         # [unix]
-    - gendict --help          # [unix]
-    - genrb --help            # [unix]
-    - icu-config --help       # [unix]
-    - icuinfo --help          # [unix]
-    - makeconv --help         # [unix]
-    - pkgdata --help          # [unix]
-    - uconv --help            # [unix]
+
+    # CLI tests
+    - derb --help              # [osx]
+    - genbrk --help            # [unix]
+    - gencfu --help            # [unix]
+    - gencnval --help          # [unix]
+    - gendict --help           # [unix]
+    - genrb --help             # [unix]
```

Comment:
Fails after this command. Not sure why though as the next command seems to work fine outside of the test step.